products/sle/main.pm

```diff
@@ -922,6 +922,26 @@ sub load_feature_tests() {
     loadtest "feature/feature_console/suseconnect.pm";
 }
 
+sub load_online_migration_tests() {
+    # stop packagekit service and more
+    loadtest "online_migration/sle12_online_migration/online_migration_setup.pm";
+    loadtest "online_migration/sle12_online_migration/register_system.pm";
```

Comment:
Why is the registration required? in the production system all of the 'source' images (SLE 12 SP1, SLE 12 GA, etc) are already registered